CONTRIBUTING

```diff
@@ -0,0 +1,43 @@
+HOW TO CONTRIBUTE TO OpenSSL
+----------------------------
+
+Development is coordinated on the openssl-dev mailing list (see
+http://www.openssl.org for information on subscribing). If you
+would like to submit a patch, send it to rt@openssl.org with
+the string "[PATCH]" in the subject. Please be sure to include a
+textual explanation of what your patch does.
+
+You can also make GitHub pull requests. If you do this, please also send
+mail to rt@openssl.org with a link to the PR so that we can more easily
```

Comment:
(fixed)

Should I myself send an updated description for the other PR I sent to the list, or is it fine for now?